database/src/migrations/release.0.2/restoration.sql

```diff
@@ -1461,566 +1784,705 @@ COMMENT ON COLUMN webform_draft.revision_count IS 'Revision count used for concu
 ;
---
--- INDEX: "Ref73136"
+-- 
+-- INDEX: "Ref253" 
 --
 
-CREATE INDEX "Ref73136" ON investment_action_category(funding_source_id)
+CREATE INDEX "Ref253" ON investment_action_category(funding_source_id)
 ;
---
--- INDEX: iucn_conservation_action_level_1_classification_nuk1
+-- 
+-- INDEX: iucn_conservation_action_level_1_classification_nuk1 
 --
 
 CREATE UNIQUE INDEX iucn_conservation_action_level_1_classification_nuk1 ON iucn_conservation_action_level_1_classification(name, (record_end_date is NULL)) where record_end_date is null
 ;
---
--- INDEX: iucn_conservation_action_level_2_subclassification_nuk1
+-- 
+-- INDEX: iucn_conservation_action_level_2_subclassification_nuk1 
 --
 
 CREATE UNIQUE INDEX iucn_conservation_action_level_2_subclassification_nuk1 ON iucn_conservation_action_level_2_subclassification(name, record_end_date, iucn_conservation_action_level_1_classification_id)
 ;
---
--- INDEX: "Ref137121"
+-- 
+-- INDEX: "Ref425" 
 --
 
-CREATE INDEX "Ref137121" ON iucn_conservation_action_level_2_subclassification(iucn_conservation_action_level_1_classification_id)
+CREATE INDEX "Ref425" ON iucn_conservation_action_level_2_subclassification(iucn_conservation_action_level_1_classification_id)
 ;
---
--- INDEX: iucn_conservation_action_level_3_subclassification_nuk1
+-- 
+-- INDEX: iucn_conservation_action_level_3_subclassification_nuk1 
 --
 
 CREATE UNIQUE INDEX iucn_conservation_action_level_3_subclassification_nuk1 ON iucn_conservation_action_level_3_subclassification(name, record_end_date, iucn_conservation_action_level_2_subclassification_id)
 ;
---
--- INDEX: "Ref139122"
+-- 
+-- INDEX: "Ref727" 
 --
 
-CREATE INDEX "Ref139122" ON iucn_conservation_action_level_3_subclassification(iucn_conservation_action_level_2_subclassification_id)
+CREATE INDEX "Ref727" ON iucn_conservation_action_level_3_subclassification(iucn_conservation_action_level_2_subclassification_id)
 ;
+-- 
+-- INDEX: linear_feature_type_nuk1 
 --
--- INDEX: permit_uk1
+
+CREATE UNIQUE INDEX linear_feature_type_nuk1 ON linear_feature_type(name, (record_end_date is NULL)) where record_end_date is null
+;
+-- 
+-- INDEX: permit_uk1 
 --
 
 CREATE UNIQUE INDEX permit_uk1 ON permit(number)
 ;
---
--- INDEX: "Ref45156"
+-- 
+-- INDEX: "Ref1317" 
 --
 
-CREATE INDEX "Ref45156" ON permit(project_id)
+CREATE INDEX "Ref1317" ON permit(project_id)
 ;
---
--- INDEX: "Ref78171"
+-- 
+-- INDEX: "Ref2926" 
 --
 
-CREATE INDEX "Ref78171" ON permit(system_user_id)
+CREATE INDEX "Ref2926" ON permit(system_user_id)
 ;
---
--- INDEX: "Ref128119"
+-- 
+-- INDEX: "Ref3023" 
 --
 
-CREATE INDEX "Ref128119" ON project(project_type_id)
+CREATE INDEX "Ref3023" ON project(project_type_id)
 ;
---
--- INDEX: project_report_attachment_uk1
+-- 
+-- INDEX: project_report_attachment_uk1 
 --
 
 CREATE UNIQUE INDEX project_report_attachment_uk1 ON project_attachment(project_id, file_name)
```

Comment:
Maybe it doesnt matter, but should this have `report` in the name?

Reply:
yep, will adjust.